### PR TITLE
Views

### PR DESCRIPTION
**Pull request by bruk19**
In this PR:-

- Added all user's records to the home page
- Implement info about users
- Added the last all records of a given user 
- Added info about posts of a given user

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,4 @@
-<h1> Here is a list of Users.</h1>
\ No newline at end of file
+<% @users.each do |use| %>
+
+  <%= render partial: 'partials/user', locals: {user: use} %>
+<% end %>
```
Comment (omar-muhamad) [inline app/views/users/index.html.erb]: - [ ] Clicking on each user supposes to take you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png), So please fix that by using `link_to` method see this [article ](https://www.rubyguides.com/2019/05/rails-link_to-method/)for more info.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,9 @@
-<h1> Here is a list of Single User. </h1>
\ No newline at end of file
+<div>
+    <%= render partial: 'partials/user', locals: {user: @user} %>
+    <p>Bio: <%= @user[0].Bio %> </p>
+
+    <p><%= @user[0].recent_post[0]%> </p>
+
+    
+    </div>
```
Comment (omar-muhamad) [inline app/views/users/show.html.erb]: - [ ] Please add the `see all posts` button and clicking on it supposes to open the user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_user_all_posts.png), So kindly fix that.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,4 @@
-<h1> Here is a list of posts</h1>
\ No newline at end of file
+<% @posts.each do |post| %>
+<%= render partial: 'partials/post', locals: {post: post} %>
+    <%= render partial: 'partials/comment', locals: {post: post} %>
+<% end %>
```
Comment (omar-muhamad) [inline app/views/posts/index.html.erb]: - [ ] Clicking on each post supposes to take you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png), So please fix that

**Review by omar-muhamad — APPROVED** [review]
# Changes Required ♻️
### Hi @bruk19,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @bruk19 , 

### Now your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## KUDOS! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**